kitsune-fe/src/components/Captcha.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <FormKit
+    type="hidden"
+    name="captchaToken"
+    validation="required"
+    v-model="captchaState.token"
+  />
+  <vue-hcaptcha
+    v-if="backend == CaptchaBackend.HCaptcha"
+    :sitekey="sitekey"
+    @verify="onVerify"
+    @expired="onExpire"
+    @error="onError"
+    @challenge-expired="onExpire"
+  >
+  </vue-hcaptcha>
+  <div v-if="backend === CaptchaBackend.MCaptcha">
+    <div id="mcaptcha__widget-container"></div>
+    <component
+      is="script"
+      type="text/javascript"
+      src="/public/mcaptcha-glue.js"
+      onload="mcaptcha__load();"
+    ></component>
+    <component is="script" type="text/javascript" charset="utf-8">
+      function mcaptcha__load() { let config = { widgetLink: new URL("{{
+        sitekey
+      }}"), }; new mcaptchaGlue.default(config); }
+    </component>
+  </div>
+</template>
+
+<script setup lang="ts">
+  import VueHcaptcha from '@hcaptcha/vue3-hcaptcha';
```

Comment:
I'm pretty sure this isn't lazy-loading the component. This means the Captcha componment is always loading the hCaptcha code, regardless of whether it is actually used.

https://vuejs.org/guide/components/async.html#basic-usage